### ISSUE TITLE
all instances: API error code 0

### ISSUE THREAD

**Issue by ghost** [pablouser1/ProxiTok#167]
Hi!
Since a few days the official instance sometimes gives the following error:
```
There was an error processing your request!

HTTP Code: 200

API error code 0 (OK)
```

**Comment by garygcchiu** [comment]
Seems fixed now, working for me again

**Comment by ggaast** [comment]
I have the same problem now. Do you think it is fixable?

**Comment by jonaskahn** [comment]
It''s broken. again :(

**Comment by pablouser1** [comment]
Fixed and explain at #179
